conda/meta.yaml

```diff
@@ -35,7 +35,7 @@ test:
 
   {# Conda does not allow spaces between package name and version, so remove them #}
   {% for package in test_dependencies %}
-    - {% if package == "graphviz" %}python-graphviz{% else %}{{ package|replace(" ", "") }}{% endif %}
+    - {% if package == "graphviz" %}python-graphviz{% elif package == "kaleido" %}python-kaleido{% else %}{{ package|replace(" ", "") }}{% endif %}
```

Comment:
If we get more examples of this, we could do something like prepend `python-` to the package name if `name in ["graphviz", "kaleido", ...]`, but this is fine for now.